scripts/boston_cityhall_electricity.json

```diff
@@ -0,0 +1,35 @@
+{
+    "citation": "FILL",
+    "description": "FILL",
+    "homepage": "FILL",
+    "keywords": [],
+    "licenses": [
+        {
+            "name": "FILL"
+        }
+    ],
+    "name": "FILL",
```

Comment:
The `name` of the dataset. This is usually the same as the file name with no extension and ` _ `replaced with ` -`  i.e `boston-cityhall-electricity`